crates/eips/src/eip1898.rs

```diff
@@ -600,12 +599,12 @@ impl fmt::Display for ParseBlockIdError {
 }
 
 #[cfg(feature = "std")]
-impl std::error::Error for ParseBlockIdError {
-    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+impl core::error::Error for ParseBlockIdError {
+    fn source(&self) -> Option<&(dyn core::error::Error + 'static)> {
         match self {
-            Self::ParseIntError(err) => std::error::Error::source(err),
-            Self::FromHexError(err) => std::error::Error::source(err),
-            Self::ParseError(err) => std::error::Error::source(err),
+            Self::ParseIntError(err) => core::error::Error::source(err),
+            Self::FromHexError(err) => core::error::Error::source(err),
```

Comment:
these are wrong btw, the source is err itself, not the source of the source

Reply:
right, updating